ddtrace/contrib/pytest/plugin.py

```diff
@@ -83,7 +83,10 @@ def pytest_addoption(parser):
 def pytest_configure(config):
     config.addinivalue_line("markers", "dd_tags(**kwargs): add tags to current span")
     if is_enabled(config):
-        _CIVisibility.enable(config=ddtrace.config.pytest)
+        try:
+            _CIVisibility.enable(config=ddtrace.config.pytest)
+        except ValueError:
+            log.warning("Failed to initialize ddtrace")
```

Comment:
What error led to this guard?

Reply:
Catching the exception here allows the test suite to continue running while civisibility is inactive, as opposed to crashing the entire program by letting the exception bubble up. The error is the one added in https://github.com/DataDog/dd-trace-py/pull/5497/commits/3f96d292330369b312eff78faaaabc0381b79177. This change is my interpretation of this recommendation: https://github.com/DataDog/dd-trace-py/pull/5497#discussion_r1170101070